deployer/scripts/upgrade.sh

```diff
@@ -46,6 +46,12 @@ function getDeploymentVersion() {
     return
   fi
 
+  # check for common data model
+  if [[ -n "$(oc get configmap/logging-elasticsearch -o yaml | grep 'use_common_data_model: true')" ]]; then
```

Comment:
Sorry I am so confused by the logic here.  It would seem on first read that for all these conditions, the first one that is hit returns at that point.  E.g., line 38, return 4 if the config map for logging-fluentd does not exist.  So if common data model is the default, then we'll return 6 all the time here, and never return the actual logging version.  Right?